### PR TITLE
Feature/mini cart

### PR DESCRIPTION
**Pull request by leticiawanderley**
### What is the purpose of this pull request?

Add mini cart component.

#### What problem is this solving?

The store user didn't have a preview of hers/his shopping cart.

#### How should this be manually tested?

`cd react`
`npm i && npm test`

#### Screenshots or example usage

screen shot 2018-04-11 at 10 39 45

screen shot 2018-04-11 at 10 40 18

#### Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/__tests__/Header.test.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ describe('Header test', () => {
   })
 
   it('should have 3 div elements', () => {
```
Comment (analuizamtg) [inline react/__tests__/Header.test.js]: "should have 4 div elements" 🙃

File: react/components/MiniCart.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  handleRemoveItem = ({ target: { id } }) => {
+    const { mutate, data: { orderForm } } = this.props
+    const itemPayload = orderForm.items.find(item => item.id === id)
+    const index = orderForm.items.indexOf(itemPayload)
+    const updatedItem = [itemPayload].map(item => {
+      return {
+        id: parseInt(item.id),
+        index: index,
+        quantity: 0,
+        seller: 1,
```
Comment (analuizamtg) [inline react/components/MiniCart.js]: hard-coded?

File: react/components/MiniCart.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        {data.orderForm.items.map(item => (
+          <div className="flex flex-row" key={item.id}>
+            <MiniCartItem
+              imageUrl={item.imageUrl}
+              name={item.name}
+              price={item.sellingPrice}
+              productId={this.getItemId(item.detailUrl)}
+            />
+            <Button id={item.id} onClick={this.handleRemoveItem}>
+              x
```
Comment (analuizamtg) [inline react/components/MiniCart.js]: tem um icone do styleguide de close, não pode usar?

File: react/components/MiniCart.js (hunk 2 above)
Comment (leticiawanderley) [reply react/components/MiniCart.js]: vai ficar assim por enquanto 😞

Comment (analuizamtg) [reply react/components/MiniCart.js]: 😲

**Pull request merged**